src/classes/handlers/PhishingHandler.ts

```diff
@@ -0,0 +1,48 @@
+import FluorineClient from '@classes/Client';
+import { readFileSync } from 'fs';
+import { fetch } from 'undici';
+export default class PhishingHandler {
+    word: string;
+    client: FluorineClient;
+    users: string;
+    constructor(client: FluorineClient) {
+        this.client = client;
+        this.users = readFileSync(
+            `${__dirname}/../../../assets/users.txt`
+        ).toString();
+        this.word = readFileSync(
+            `${__dirname}/../../../assets/words.txt`
+        ).toString();
```

Comment:
```suggestion
        );
```

Reply:
if you dont put in .toString() im pretty sure ts returns an error